src/app/common/stores/applications/SsvWeb/OperatorMetadata.store.ts

```diff
@@ -119,8 +119,10 @@ class OperatorMetadataStore extends BaseStore  {
             let value = this.getMetadataValue(field);
             if (field === FIELD_KEYS.MEV_RELAYS && typeof value !== 'string') {
                 value = value.join(',');
+            } else if (field === FIELD_KEYS.DKG_ADDRESS && value === HTTP_PREFIX) {
```

Comment:
Value should be trimmed, better in `getMetadataValue`. This logic won't work if user will enter empty space